app/controllers/ExclusionListController.scala

```diff
@@ -164,40 +164,65 @@ class ExclusionListController @Inject()(
               } yield {
                 selectedValue match {
                   case ControllersReferenceDataCodes.FORM_TYPE_NINO =>
-                    Ok(
-                      ninoExclusionSearchFormView(
-                        taxYearRange,
-                        isCurrentTaxYear,
-                        iabdTypeValue,
-                        formMappings.exclusionSearchFormWithNino,
-                        empRef = request.empRef))
+                    Redirect(
+                      routes.ExclusionListController.showExclusionSearchForm(isCurrentTaxYear, iabdType, "nino")
+                    )
                   case ControllersReferenceDataCodes.FORM_TYPE_NONINO =>
-                    Ok(
-                      noNinoExclusionSearchFormView(
-                        taxYearRange,
-                        isCurrentTaxYear,
-                        iabdTypeValue,
-                        formMappings.exclusionSearchFormWithoutNino,
-                        empRef = request.empRef))
-                  case "" =>
                     Redirect(
-                      routes.ExclusionListController.withOrWithoutNinoOnPageLoad(isCurrentTaxYear, iabdTypeValue))
+                      routes.ExclusionListController.showExclusionSearchForm(isCurrentTaxYear, iabdType, "no-nino"))
+                  case "" =>
+                    Redirect(routes.ExclusionListController.withOrWithoutNinoOnPageLoad(isCurrentTaxYear, iabdType))
                       .flashing("error" -> Messages("ExclusionDecision.noselection.error"))
                 }
               }
             }
           )
         controllersReferenceData.responseErrorHandler(resultFuture)
       } else {
+        Logger.info("[ExclusionListController][withOrWithoutNinoDecision] Exclusions not allowed, showing error page")
         Future.successful(
-          Ok(
+          MethodNotAllowed(
```

Comment:
Not sure about this normally the 405 status code is for doing something like a POST on a GET, I think a Forbidden would be better suited for this, happy to discuss this. [mozilla http status](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)

Reply:
Reading that documentation I definitely agree - I used quite a few of these 405s so will update them all to 403s